core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-properties/dot-container-properties.component.html

```diff
@@ -1,53 +1,95 @@
+            <p-inplace #editText>
+                <ng-template pTemplate="content">
+                    <span class="p-input-icon-right">
+                        <i class="pi pi-times" (click)="editText.deactivate()"></i>
+                        <input type="text" pInputText formControlName="title" />
+                    </span>
+                </ng-template>
+                <ng-template pTemplate="display">
+                    <span
+                        class="dot-container-properties__title-text"
+                        *ngIf="!form?.value?.title"
+                        >{{ 'message.containers.create.click_to_edit' | dm }}</span
+                    >
+                    <span class="dot-container-properties__title-text" *ngIf="form?.value?.title">{{
+                        form?.value?.title
+                    }}</span>
+                    <i class="pi pi-pencil"></i>
+                </ng-template>
+            </p-inplace>
+        </div>
+        <div class="field">
+            <label class="block" for="description">{{
+                'message.containers.create.description' | dm
+            }}</label>
+            <input id="description" type="text" formControlName="friendlyName" pInputText />
+        </div>
+        <div class="field">
+            <label class="block" for="max-contents">{{
+                'message.containers.create.max_contents' | dm
+            }}</label>
+            <input id="max-contents" type="text" formControlName="maxContentlets" pInputText />
+            <button
+                class="p-button-secondary"
+                *ngIf="vm.isContentTypeVisible"
+                [label]="'message.containers.create.clear' | dm"
+                (click)="clearContent()"
+                pButton
+                type="button"
+            ></button>
+            <button
+                *ngIf="!vm.isContentTypeVisible"
+                [label]="'message.containers.create.content_type_code' | dm"
+                [disabled]="!vm.isContentTypeButtonEnabled"
+                (click)="showContentTypeAndCode()"
+                pButton
+                type="button"
+            ></button>
+        </div>
 
-    <div class="dot-container-properties__code-container-outer">
-        <form [formGroup]="form">
-            <dot-loop-editor
-                (buttonClick)="showLoopInput()"
-                [isEditorVisible]="vm.showPrePostLoopInput"
-            ></dot-loop-editor>
-            <dot-content-editor></dot-content-editor>
-            <dot-loop-editor
-                (buttonClick)="showLoopInput()"
-                [isEditorVisible]="vm.showPrePostLoopInput"
-            ></dot-loop-editor>
-        </form>
-    </div>
+        <div class="dot-container-properties__code-container-outer">
+            <div *ngIf="vm.isContentTypeVisible">
+                <dot-loop-editor
+                    [isEditorVisible]="vm.showPrePostLoopInput"
+                    [control]="$any(form).get('preLoop')"
```

Comment:
https://angular.io/guide/template-expression-operators#any-type-cast-function

Looks like this is going away, please use something else to avoid problems in the future.